.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+##### Description
+
+<!---
+Please include a short description of what your PR does and / or the motivation behind it
+--->
+
+##### Checklist
+
+- [ ] Built with `cargo build`
```

Comment:
(Apologies I am going to be a bit of a stickler here since I want to make sure we don't ask contributors to do anything more than necessary)

I don't think they need to do this, `nix build` will do it for them and I imagine most folks will have editor tooling doing error checks.